src/matchers/toBeFrozen/index.js

```diff
@@ -0,0 +1,18 @@
+import { matcherHint } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = () => () => matcherHint('.not.toBeFrozen') + '\n\n' + 'Expected object to not be frozen';
```

Comment:
The first empty function (`() =>`) can be removed and then you don't need to invoke it below :smile:

If you change the `matcherHint` to be `matcherHint('.not.toBeFrozen', 'recieved', '')` it will change the output message from:

```
expect(recieved).not.toBeFrozen(expected)
```

to:
```
expect(recieved).not.toBeFrozen()
```